.github/workflows/ci.yaml

```diff
@@ -155,6 +151,48 @@ jobs:
           docker build -f docker/pmbgpd/Dockerfile -t pmbgpd:_build .
           docker build -f docker/pmbmpd/Dockerfile -t pmbmpd:_build .
           docker build -f docker/pmtelemetryd/Dockerfile -t pmtelemetryd:_build .
+          echo "Saving images as artifacts..."
+          mkdir -p /tmp/docker/
+          docker save -o /tmp/docker/pmacct_docker_images.tar base:_build pmacctd:_build nfacctd:_build sfacctd:_build uacctd:_build pmbgpd:_build pmbmpd:_build pmtelemetryd:_build
+
+      - name: Export pmacct docker images as an artifact
+        uses: actions/upload-artifact@v2.2.2
```

Comment:
im an ignorant regarding this, but: is there any chance that two parallel PRs conflict with each other / replace the images we are saving in this artifact storage resource? I want to think that something already adds versioning for you, so one pipeline execution does not reuse the artifacts of another build, but just raising the hand, in case we are missing something

Reply:
https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts

Artifacts are per Github Actions workflow (similarly, say in Gitlab artifacts are per pipeline).